src/VideoStreaming/VideoReceiver.cc

```diff
@@ -468,6 +474,73 @@ VideoReceiver::start()
 #endif
 }
 
+#if defined(QGC_GST_STREAMING)
+void
+VideoReceiver::_startAudio()
+{
+    // Run audio
+    uint32_t port = _videoSettings->audioUdpPort()->rawValue().toUInt();
+
+    QString pipeline = QStringLiteral("udpsrc port=%1 "
+        "! application/x-rtp, media=audio, clock-rate=44100, encoding-name=L16, encoding-params=1, channels=1, payload=96 "
+        "! rtpL16depay name=rtp0 "
+        "! audioconvert "
+        "! volume volume=1.0 name=volume0 "
+        "! queue "
+        "! autoaudiosink sync=false").arg(port);
+
+    qCDebug(VideoReceiverLog) << "Opening audio pipeline: " << pipeline;
+
+    _audioPipeline = gst_parse_launch(static_cast<char*>(pipeline.arg(port).toLocal8Bit().data()) , nullptr);
+
+    if (_audioPipeline) {
+        gst_element_set_state(_audioPipeline, GST_STATE_PLAYING);
+
+        _gstVolume = gst_bin_get_by_name(GST_BIN(_audioPipeline), "volume0");
+        qCDebug(VideoReceiverLog) << "volume0:" << _gstVolume;
+        setVolume(_videoSettings->audioVolume()->rawValue().toFloat());
+    }
```

Comment:
We should print an error message if it fails.

Reply:
advice still applies

Reply:
Do you mean a `qgcApp()->showMessage()` ?

Reply:
```suggestion
    } else {
        qCDebug(VideoReceiverLog) << "Failed to get audio pipleine"
    }
```
or similar

Reply:
I don't expect both `_audioPipeline` and `error` to be null at the same time.
isn't [line 497](https://github.com/mavlink/qgroundcontrol/pull/7188/files#diff-ff57b92b141a56c61545a49ceedbe906R497) enough?

Reply:
Done.
I expected that if `error` was set then `_audioPipeline` would be `NULL`, but looking again at the docs that is not always the case, my bad.